### PR TITLE
use logverification event structures

### PR DESCRIPTION
**Pull request by honourfish**
Draft

re: AB#9604

### CODE REVIEW

File: readeventsfile.go [hunk 1]
```diff
@@ -32,12 +33,27 @@ func readArgs0FileOrStdIo(cCtx *cli.Context) ([]VerifiableEvent, error) {
 	return VerifiableEventsFromData(data)
 }
 
+func readArgs0FileOrStdIoToDecodedEvent(cCtx *cli.Context) ([]logverification.DecodedEvent, error) {
```
Comment (henry739) [inline readeventsfile.go:36]: This one too

File: readeventsfile.go [hunk 2]
```diff
@@ -55,17 +71,45 @@ func VerifiableEventsFromData(data []byte) ([]VerifiableEvent, error) {
...
+
 	return verifiableEvents, nil
 }
 
+// ReadDecodedEventsFromFile reads datatrails events from a file and returns a
+// normalized list of raw binary items.
+//
+// See EventListFromData for the content expectations (must be a list of events
+// or single event from datatrails api)
+func ReadDecodedEventsFromFile(fileName string) ([]logverification.DecodedEvent, error) {
```
Comment (henry739) [inline readeventsfile.go:87]: If we're reading events from a file, I think its kind of implied that they're decoded. Given that the function signature tells us its a DecodedEvent, I think we can drop this from the name.

File: readeventsfile.go [hunk 3]
```diff
@@ -6,18 +6,19 @@ import (
...
-func readArgs0File(cCtx *cli.Context) ([]VerifiableEvent, error) {
+func readArgs0File(cCtx *cli.Context) ([]logverification.VerifiableEvent, error) {
 	if cCtx.Args().Len() < 1 {
 		return nil, fmt.Errorf("filename expected as first positional command argument")
 	}
 	return ReadVerifiableEventsFromFile(cCtx.Args().Get(0))
 }
 
-func readArgs0FileOrStdIo(cCtx *cli.Context) ([]VerifiableEvent, error) {
+func readArgs0FileOrStdIoToVerifiableEvent(cCtx *cli.Context) ([]logverification.VerifiableEvent, error) {
```
[the remark this answers is not in the document]
Comment (honourfish) [reply readeventsfile.go:21]: ahah yes :) sorry just did the fastest thing to get it working

**Review by henry739 — APPROVED** [review]

**Comment by honourfish** [comment]
Naming comments will be addressed in a subsequent PR

**Pull request merged**